Packs/Gmail/Integrations/Gmail/Gmail.py

```diff
@@ -2129,62 +2129,58 @@ def send_mail(emailto, emailfrom, subject, body, entry_ids, cc, bcc, htmlBody, r
 
 def send_mail_command():
     args = demisto.args()
-    emailto = argToList(args.get('to'))
-    emailfrom = args.get('from')
+    return mail_command(args)
+
+
+def mail_command(args, subject_prefix='', in_reply_to=None, references=None):
```

Comment:
Can this method be extracted to GmailApiClient or something?
It seems pretty close to the one in GmailSingleUser

Reply:
@tkatzir  gmail doesn't have a shared module, this could of saved a lot of duplicate code. i think it should be out of the scope of this task

Reply:
Agree.
If you think there's more "meat" for a GmailApiModule, then open a dev-task.
Otherwise, we'll leave it as is.

Reply:
there is, ticket created